solc/CommandLineParser.h

```diff
@@ -203,6 +203,25 @@ class CommandLineParser
 	bool hasOutput() const { return m_hasOutput; }
 
 private:
+	/// Returns a specification of all named command-line options accepted by the compiler.
+	/// The object can be used to parse command-line arguments or to generate the help screen.
+	static boost::program_options::options_description optionsDescription();
+
+	/// Returns a specification of all possitional command-line arguments accepted by the compiler.
+	/// The object can be used to parse command-line arguments or to generate the help screen.
+	static boost::program_options::positional_options_description positionalOptionsDescription();
```

Comment:
```suggestion
	static boost::program_options::positional_options_description _positionalOptionsDescription();
```

Reply:
This one is actually a function, not an argument :)

Reply:
haha I went overboard